Language/Expressions/Constants/Constant_Contexts/constant_context_A02_t01.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Let e be an expression; e occurs in a constant context iff one of
+/// the following applies:
+/// ...
+/// • e occurs as @e in a construct derived from ⟨metadata⟩.
+///
+/// @description Checks that an expression `e` is a constant expression if `e`
+/// occurs as `@e` in a construct derived from ⟨metadata⟩
+/// @author sgrekhov22@gmail.com
+
+import 'dart:mirrors';
```

Comment:
'dart:mirrors' will not work in the VM AOT mode, this test needs to be skipped for AOT builds.

Also 'dart:mirrors' is an unsupported feature, not sure why we need to add new tests to test it's funcitonality

Reply:
@a-siva thank you! https://dart-review.googlesource.com/c/sdk/+/353522